src/components/progress-bar/progress-bar.stories.ts

```diff
@@ -0,0 +1,39 @@
+export default {
+  title: 'Components/Progress Bar',
+  args: {
+    size: "m",
+    percentage: 50,
+    showLabel: false,
+
+  },
+  argTypes: {
+    percentage: {
+      control: { type: 'range', min: 0, max: 100 },
+    },
+    size: {
+      options: ['s', 'm'],
+      control: { type: 'radio' },
+    },
+  },
+};
+
+const DefaultTemplate = (args) =>
+  `<ifx-progress-bar percentage="${args.percentage}" size="${args.size}" show-label="${args.showLabel}" label="${args.percentage}%"></ifx-progress-bar>`;
+
+export const Default = DefaultTemplate.bind({});
+Default.args = {
+};
+
+export const Small = DefaultTemplate.bind({});
+Small.args = {
+  percentage: 75,
+  size: 's',
+
+};
```

Comment:
I think it's better to either not have variants for `small` and `with label`, and only have these as
control options, or vice versa, only variants, and no control options. Having both is unnecessary.